dev-lang/python/python-3.13.0_rc1_p3.ebuild

```diff
@@ -125,6 +126,14 @@ pkg_pretend() {
 		ewarn "you can reproduce the problem with dev-lang/python[gil,-jit].  Instead,"
 		ewarn "please consider reporting freethreading / JIT problems upstream."
 	fi
+
+	if use kernel_linux && [[ "${MERGE_TYPE}" != "binary" ]] && use test; then
+		linux-info_pkg_setup
+		require_configured_kernel
+		local CONFIG_CHECK="DNOTIFY" # bug 938662
+		CONFIG_CHECK+=" CROSS_MEMORY_ATTACH" # bug 938589
+		check_extra_config
+	fi
```

Comment:
I still think this is invalid. Remember: we're not building kernel modules, there is no reason to force to build to fail if there's no configured kernel available or if the configuration doesn't match the currently running kernel.